src/data.js

```diff
@@ -2398,6 +2398,29 @@ module.exports = [
       'Blogger',
     ],
   },
+  {
+    name: 'Jenna Pederson',
+    description:
+      'technical entrepreneur with a passion for community and equity-building',
```

Comment:
A little nitpicky, but could you make this start with an uppercase T? Though the checks are "failing", nothing is actually wrong with this, so once that's done we can merge. Thanks!